webapp/pages/login.vue

```diff
@@ -10,9 +10,6 @@
       <ds-card class="login-card">
         <ds-flex gutter="small">
           <ds-flex-item :width="{ base: '100%', sm: '50%' }" centered>
-            <client-only>
-              <locale-switch class="login-locale-switch" offset="5" />
-            </client-only>
```

Comment:
@Tirokk Thanks for the feedback - are you suggesting that we remove the header from all the pages that are now using the new basic layout?  If so we would need to consider where we place the language selector on some of the other pages such as the Imprint, Terms and Conditions, Data Privacy... pages?  Or should we keep the basic layout for these pages and create a new blank layout for the Login and possibly password reset pages?

Reply:
I think the idea in https://github.com/Human-Connection/Human-Connection/pull/1283#issuecomment-523478461 was to remove the top bar *only* from the login page, but keep the top bar on the pages linked in the footer (code of conduct etc). Maybe you could use the `blank` layout and add the footer manually?

Reply:
Oh and http://localhost:3000/registration/create-user-account and http://localhost:3000/password-reset/request would have the same layout as login. The language chooser is missing there in the component, I guess (you could add that probably :wink: )

Reply:
@roschaefer & @Tirokk I've now updated the Login, Password Reset and Registration pages so that the header has been removed.  The other pages such as Imprint, terms and conditions etc continue to use the `basic` layout.  On my development machine the logo is in line with the content on these pages.

Reply:
![Registration Page](https://dl.dropbox.com/s/aknsgu1cckiqmdm/Screenshot%202019-09-07%2013.16.32.png?dl=0)

This is how the registration pages displays locally for me now with the latest changes 👆